### ISSUE TITLE
Implement Email Automations

### ISSUE THREAD

**Issue by wmnnd** [pentacent/keila#132]
Keila should allow the creation of email automations and drip/style campaigns.

This requires two main components:
- Triggers: An entity that defines when an automation (or a part of an automation) should be executed. Triggers may include things like "contact has subscribed", "contact has clicked on an email link", or "contact has subscribed 14 days ago".
- Automated campaigns. Automated campaigns, unlike regular campaigns, are not sent to all contacts (or a segment), but are instead sent to contacts individually, as a result of a trigger being executed.

More ideas are welcome!

**Comment by satoshinotdead** [comment]
What about using n8n? They can handle [get/post webhooks](https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.webhook/#webhook-urls) and [http requests](https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.httprequest).

**Comment by miroslavkosut** [comment]
Hello everyone, in my use case, this is the most-wanted feature. Please take a look how [loops.so](loops.so) is doing it for inspiration. It would be great to have an open source email automation.

**Comment by dshook** [comment]
Plus one to this feature.  Even just starting with a way to automatically send a sequence to new signups would be great before going down the rabbit hole of all the different ways to trigger and set up automations

**Comment by oliver3** [comment]
We would be using it for an email course that you can sign up for, and you'll receive a new lesson every week.

**Comment by Lislegaard** [comment]
This would be super useful! As a music artist I imagine things like get someone to sign up for the newsletter then have 2-3 emails spread out over a couple of months introducing other arenas to check out. Like a Bandcamp site, Owncast site and 1-2 favorite blog posts. A way to give information without bringing too much noise at the same time.

Also it could work well for (in lack of a better word) unlocking content. "Sign up to my newsletter to get a free download of my last live recording".

**Comment by jrgong420** [comment]
Exactly what I am looking for, +1 for this

**Comment by toineenzo** [comment]
Any news on this?

**Comment by wmnnd** [comment]
Yes, I'm currently making good progress on #537 and after that, automations will be the next big feature.
